test/distributed/cases/mo_cloud/mo_cloud.result

```diff
@@ -597,9 +597,8 @@ information_schema    16    accountadmin
 mysql    5    accountadmin
 mo_mo    0    accountadmin
 mo_catalog    19    -
-SELECT mo_catalog.mo_database.datname,mo_catalog.mo_tables.relname,mo_catalog.mo_tables.relkind, if (role_name IS NULL,'-', role_name) AS `owner` FROM mo_catalog.mo_database LEFT JOIN mo_catalog.mo_tables ON mo_catalog.mo_database.datname = mo_catalog.mo_tables.reldatabase LEFT JOIN mo_catalog.mo_role ON mo_catalog.mo_tables.owner=role_id ORDER BY reldatabase, relname;
+SELECT mo_catalog.mo_database.datname,mo_catalog.mo_tables.relname,mo_catalog.mo_tables.relkind, if (role_name IS NULL,'-', role_name) AS `owner` FROM mo_catalog.mo_database LEFT JOIN mo_catalog.mo_tables ON mo_catalog.mo_database.datname = mo_catalog.mo_tables.reldatabase LEFT JOIN mo_catalog.mo_role ON mo_catalog.mo_tables.owner=role_id WHERE relname NOT LIKE "__mo_index_secondary_%" ORDER BY reldatabase, relname;
 datname    relname    relkind    owner
-mo_mo    null    null    -
```

Comment:
Need to analyse why this row is gone when I added `WHERE relname NOT LIKE "__mo_index_secondary_%"` condition.

Reply:
In SQL, when you evaluate a column with a NULL value using the function LIKE, the result is NULL. In a WHERE clause, a condition that evaluates to NULL will not be considered true, so the row will not be included in the result set.

Fix: https://stackoverflow.com/a/5910470/1609570